README.md

```diff
@@ -25,48 +26,47 @@ The prefered way to install HOMS is to use [Docker](https://www.docker.com/).
-2. Download `docker-compose.yml`:
+1. Clone this repo and navigate to its directory:
 
     ```bash
-    wget https://raw.githubusercontent.com/latera/homs/master/docker-compose.yml
+    git clone git@github.com:latera/homs.git && cd homs
     ```
 
-3. For OS X users: make path to folder with HOMS shared in `Docker -> Preferences... -> File Sharing`.
+2. For OS X users: make path to folder with HOMS shared in `Docker -> Preferences... -> File Sharing`.
 
-4. Copy your (or default) `.env` file to your project's directory:
+3. Copy your (or default) `.env` file to your project's directory:
 
     ```bash
     cp .env.sample .env
     ```
 
     :pushpin: All variables are set in `.env` file. There you can change them, if you want to.
 
-5. Set `HOMS_PATH` variable to your `.env` with path to your HOMS source directory.
+4. Set `HOMS_PATH` variable to your `.env` with path to your HOMS source directory.
 
-6. Add `SECRET_KEY_BASE` variable to your `.env` with uniq id as value. You can generate key with `openssl rand -hex 64` command. For example:
+5. Add `SECRET_KEY_BASE` variable to your `.env` with uniq id as value. You can generate key with `openssl rand -hex 64` command. For example:
 
     ```bash
     SECRET_KEY_BASE=0750fd0eac13032778f0a42e2ab450003eaece477ea881501be0cc438f870a2f498dbbc00ffb7c8379c30c960568a402d315496bb7bc2b3ee324401ba788a
     ```
 
     :warning: Make sure this key is secret and don't share it with anyone.
 
-7. Change [Minio](https://github.com/minio/minio) credentials in `.env` file. Generate `MINIO_ACCESS_KEY` and `MINIO_SECRET_KEY` values with any credentials generator, e.g. `pwgen 32 2`.
+6. Change [Minio](https://github.com/minio/minio) credentials in `.env` file. Generate `MINIO_ACCESS_KEY` and `MINIO_SECRET_KEY` values with any credentials generator, e.g. `pwgen 32 2`.
 
-8. Run `docker-compose`:
+7. Run `docker-compose`:
 
     ```bash
     docker-compose up -d
     ```
 
-9. Navigate to [Minio control panel](http://localhost:9000) and create a bucket with name equal to `MINIO_BUCKET_NAME` value from `.env` file.
+8. Navigate to [Minio control panel](http://localhost:9000) and create a bucket with name equal to `MINIO_BUCKET_NAME` value from `.env` file.
 
-10. Navigate to [Camunda Admin](http://localhost:8766/camunda) and create admin user with credentials equal to `BPM_USER:BPM_PASSWORD` values from `.env` file.
+9. Navigate to [Camunda Admin](http://localhost:8766/camunda) and login with credentials equal to `BPM_USER:BPM_PASSWORD` values from `.env` file (`user/changeme` if these variables is not set).
 
-11. (Optional) If you want to use demo processes, navigate to [Camunda](http://localhost:8766/camunda/app/admin/default/#/user-create) and create user with `user@example.com` email.
+10. [Create user in Camunda](http://localhost:8766/camunda/app/admin/default/#/user-create) with `user@example.com` email (or change email of existent user).
```

Comment:
Actually, this is already done by [seed application](https://github.com/latera/camunda-ext/blob/master/seed/src/main/groovy/org/camunda/latera/bpm/configuration/SeedApplication.groovy#L51). This step can be removed

Reply:
Well, OK but I had no chance to know this, it isn't reflected in `camunda-ext` readme.